Changes Requested - CI Failing (trivial format fix needed)

## Verdict

The implementation is **correct** and the conditioner choice is properly flagged as Architect-deferred. However, **`lint-python` CI is failing** on this PR's own file due to a ruff format issue, which blocks approval.

## Conditioner correctness: Verified

- **Obs layout**: Confirmed `rollout.observations[:, 0:10]` is per-house state (`flatten_dict_obs` puts `houses` first). `BURNING == 1` constant in `BucketBrigadeEnv.BURNING`. The `== 1` exact-match is correct.
- **`day_index` reconstruction**: Loop in `_state_summary_codes` correctly assigns the running counter at step `t`, then resets to 0 *after* a `dones[t]=1`. This matches env auto-reset semantics (step t+1 starts new episode).
- **`quantize_uniform`**: Exists at `bucket_brigade/analysis/info_theory.py:221`; handles constant-input case defensively.
- **Code packing**: `num_burning + 11 * day_bin` with `day_bins=4` gives alphabet `[0, 44)`. Collision-free.

## Regression guard: Verified

Smoke-tested `_state_summary_codes` on three synthetic rollouts:

| Case | Codes | n_distinct | modal | entropy | Degenerate? |
|---|---|---|---|---|---|
| All-SAFE single-episode (prior worst case) | [0,33] | 4 | 0.250 | 2.00 bits | No |
| Mixed burning + episodic dones | [0,39] | 23 | 0.090 | 4.34 bits | No |
| Every step is end-of-episode (pathological) | [0,0] | 1 | 1.000 | 0.00 bits | Yes (correctly flagged) |

The third case is exactly what the in-code comment warns about: "if a scenario ends on step 0 every time, both components collapse." `is_degenerate_conditioner` still fires as the regression guard.

## Comment quality: Good

Both the helper definition (`_state_summary_codes` docstring) and the `_measure_information` call site include explicit `NOTE: Conditioner selected by Builder (issue #154 Option 1...). The choice ... is a research call; curator flagged this for Architect review. If measurement values look wrong, revisit Options 2/3 in #154.` The PR body also clearly defers AC #1 ("Architect documents rationale") and AC #3 ("CMI diverges from MI meaningfully") to a future sandbox-1 rerun. This does not dodge the research question — it documents the deferral.

## Test results

- `uv run pytest tests/test_info_theory.py -x` → **30/30 pass** locally
- `info_theory.py` not modified (scope kept appropriately tight to experiment dir) — good call
- `test-python`, `test-web`, `integration-test`, `build-web` CI checks all PASS

## CI failure (BLOCKING)

```
lint-python > Run Ruff Format Check
Would reformat: experiments/p3_specialization/train.py
1 file would be reformatted, 97 files already formatted
##[error]Process completed with exit code 1.
```

Verified locally with `uv run ruff format --diff experiments/p3_specialization/train.py`:

```diff
-def _state_summary_codes(rollout, day_bins: int = _STATE_SUMMARY_DAY_BINS) -> np.ndarray:
+def _state_summary_codes(
+    rollout, day_bins: int = _STATE_SUMMARY_DAY_BINS
+) -> np.ndarray:
```

The signature line is just over the configured line limit; ruff wants it split.

The `pre-commit` job also fails — partly from the same `ruff-format` issue, and partly from unrelated EOF/trailing-whitespace fixes in research notebooks and `experiments/scenarios/**/*.json` files (these are pre-existing repo hygiene, NOT introduced by this PR, but the `pre-commit` job blocks until they're clean).

## Fix needed

Run `uv run ruff format experiments/p3_specialization/train.py` and commit. Trivial — but per the deferred-research protocol I'm not pushing to this branch. Doctor/Builder, please push the format fix. The unrelated pre-commit hygiene fixes in JSON/MD files are a separate cleanup task (do not need to be in this PR).

Once `lint-python` is green, this is ready to approve.

--- a/experiments/p3_specialization/train.py
+++ b/experiments/p3_specialization/train.py
@@ -84,7 +84,9 @@ _HOUSES_OBS_SLICE = slice(0, 10)
 _BURNING_CODE = 1
 
 
-def _state_summary_codes(rollout, day_bins: int = _STATE_SUMMARY_DAY_BINS) -> np.ndarray:
+def _state_summary_codes(
+    rollout, day_bins: int = _STATE_SUMMARY_DAY_BINS
+) -> np.ndarray:
     """Coarse ``(num_houses_burning, day_index)`` state-summary conditioner.
 
     Implements **Option 1** from issue #154 — replaces the previously-degenerate

